pkg/kubelet/dockertools/manager.go

```diff
@@ -1392,10 +1392,6 @@ func (dm *DockerManager) killContainer(containerID kubecontainer.ContainerID, co
 		gracePeriod = minimumGracePeriodInSeconds
 	}
 	err := dm.client.StopContainer(ID, uint(gracePeriod))
-	if _, ok := err.(*docker.ContainerNotRunning); ok && err != nil {
```

Comment:
Because the `ContainerStop` function won't return error anymore, if the container is not running. Here is related code in [Docker 1.8.0](https://github.com/docker/docker/blob/v1.8.0/daemon/container.go#L494.) (The minimum version we support)